examples/one_dot_one/src/generatedNoCheck/objects/ObjectTypeWithAllPropertyTypes.ts

```diff
@@ -0,0 +1,142 @@
+import { ObjectDefinition } from '@osdk/api';
+import type {
+  OntologyObject,
+  LocalDate,
+  Timestamp,
+  GeoShape,
+  GeoPoint,
+  Attachment,
+  TimeSeries,
+} from '@osdk/legacy-client';
+
+/**
+ * A type with all property types
+ */
+export interface ObjectTypeWithAllPropertyTypes extends OntologyObject {
```

Comment:
Just for illustrative purposes, can remove this one before we go forth with anything